components/brave_news/browser/resources/internals.html

```diff
@@ -14,7 +14,7 @@
   <style>
     body {
       overflow-x: hidden;
-      background: var(--leo-gradient-toolbar-background);
+      background: var(--leo-color-page-background);
```

Comment:
nit as it's internal page.
text seems not visible well with bg color in dark theme.
It it's not intended I think it would be better to delete `background: rgba(255, 255, 255, 0.7);` from `Grid` component.
<img width="621" alt="Screenshot 2024-08-23 at 4 22 49 PM" src="https://github.com/user-attachments/assets/30da9bd4-8745-4848-ae77-90e3a45c2c9d">

Reply:
Good call!